### DISCUSSION TITLE
png quality

### DISCUSSION THREAD

**Discussion by graz68a** [lllyasviel/Fooocus#2791] [Q&A] [answered]
Hello

does exist any way to increase .png quality in Fooocus ? It (default) seems set in a low/medium quality .

Thank you

**Comment by mashb1t** [comment, marked as answer]
Fooocus does not reduce PNG quality and keeps jpeg and webp at 95%, see https://github.com/lllyasviel/Fooocus/blob/main/modules/private_logger.py#L42 and below. Which measurements indicate to you that PNG quality seems to be set to low/medium?
